docs/content/guides/major-pg-upgrade.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "PostgreSQL Major Upgrade"
+date:
+draft: false
+weight: 100
+---
+
+You can perform a major PostgreSQL upgrade from one version to another using PGO. Described below are the 
+steps necessary to perform your upgrade, which uses the 
+[pg_upgrade](https://www.postgresql.org/docs/current/pgupgrade.html) tool.
+
+{{% notice warning %}}
+Please note that your PostgresCluster will need to be in a healthy state in order for the upgrade to 
+complete as expected. If there are any present issues, such as Pods that are not running correctly or 
+other similar problems, they will need to be addressed before proceeding!
+{{% /notice %}}
+
+### Step 1: Take a pgBackRest Backup
+
+Before starting your major upgrade, you should take a new database 
+[backup]({{< relref "tutorial/backup-management.md" >}}). This will offer another layer of data 
+protection in cases where the upgrade process does not complete as expected.
+
+### Step 2: Scale Down Replicas
+
+PGO needs to identify the primary database instance before the running your upgrade because the primary
+pgdata volume will need to be identified and mounted to the upgrade Job. Any cluster replicas will not be
```

Comment:
```suggestion
PGO needs to identify the primary database instance before running your upgrade because the primary
pgdata volume will need to be mounted to the upgrade Job. Any cluster replicas will not be
```